ansible/roles/schulcloud-server-core/templates/api-h5p-svc.yml.j2

```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: {{ NAMESPACE }}
   labels:
     app: api-h5p
-  annotations:
-    prometheus.io/scrape: 'true'
-    prometheus.io/port: '4448'
```

Comment:
It's fine, but please make you an follow up ticket where you add the new metric endpoint to this service.